og_ui/og_ui.services.yml

```diff
@@ -0,0 +1,6 @@
+services:
+  access_check.og.user_access_group:
```

Comment:
The _historical_ reason `subscribe`/ `subscribe without approval`/ `unsubscribe` are part of OG-UI and not OG, is that it kind of feels like subscription has a dependency on a UI. It's more clear when we think about the permission "subscribe" as one that requires a group-admin confirmation - form me it implies there's a UI there, and it's not on the API level.

Anyway, maybe it could also move to OG itself, because _now days_, when I think of it, it feels more related to the API. Maybe I became wiser? :P

> so other people can build up their own UI around OG?

I'd love to see that, but I don't think it will happen. Or at least I haven't seen it even once :wink: